src/repository.rs

```diff
@@ -194,7 +190,8 @@ impl Repository {
     pub fn check_stream(&self, sha256: &Sha256HashValue) -> Result<Option<Sha256HashValue>> {
         match self.openat(&format!("streams/{}", hex::encode(sha256)), OFlags::RDONLY) {
             Ok(stream) => {
-                let measured_verity: Sha256HashValue = fs_ioc_measure_verity(&stream)?;
+                let measured_verity: Sha256HashValue = fs_ioc_measure_verity(&stream)?
```

Comment:
Low-key surprised to see that we legit only had a single case where "option" wasn't the right API.

Reply:
On the other hand, this seems to be the only API user that evaded the 'compare'-ification.  I wonder if `Option` on the base API was wrong here, after all.  Hm.

Reply:
I added an intermediate function which requires verity